### PR TITLE
task_DEV-5

### PR DESCRIPTION
**Pull request by VadimBelkin**

### CODE REVIEW

File: IncreasingSequence.java [hunk 1]
```diff
@@ -2,20 +2,28 @@
...
+    } else {
+      Scanner scan = new Scanner(System.in);
+      System.out.print("Enter the numbers through a space: ");
+      String sequence = scan.nextLine();
+      String sequenceParts[] = sequence.split(" ");
+      nondecreasing(sequenceParts, sequenceParts.length);
+    }
+  }
+
+public static void nondecreasing(String []sequence, int length) {
```
Comment (AlexanderSoroka) [inline IncreasingSequence.java]: missed javadoc comment - codestyle violation (7.3)

Comment (AlexanderSoroka) [inline IncreasingSequence.java]: Do you really need to pass array length separately? Arrays in java have field .length

File: IncreasingSequence.java [hunk 2]
```diff
@@ -2,20 +2,29 @@
...
+      System.out.print("Enter the numbers through a space: ");
+      String sequence = scan.nextLine();
+      String sequenceParts[] = sequence.split(" ");
+      nondecreasing(sequenceParts);
     }
-   }
- }
+  }
+
+/** Determines which sequence and displays information about this on the screen. */
```
Comment (AlexanderSoroka) [inline IncreasingSequence.java]: javadoc violation

File: IncreasingSequence.java [hunk 3]
```diff
@@ -2,20 +2,29 @@
...
+public static void nondecreasing(String []sequence) {
```
Comment (AlexanderSoroka) [inline IncreasingSequence.java]: codestyle violation

File: IncreasingSequence.java (hunk 2 above)
Comment (VadimBelkin) [inline IncreasingSequence.java]: I don't understand, what to change in the javadoc? In 7.2 there is an example. Or it not that? Help me, please.

Comment (AlexanderSoroka) [inline IncreasingSequence.java]: it cannot be one-line comment, all methods argument must be described 
http://www.oracle.com/technetwork/articles/java/index-137868.html

Comment (VadimBelkin) [inline IncreasingSequence.java]: Thank you very much!

File: IncreasingSequence.java [hunk 4]
```diff
@@ -2,20 +2,37 @@
...
+      System.out.print("Enter the numbers through a space: ");
+      String sequence = scan.nextLine();
+      String sequenceParts[] = sequence.split(" ");
+      nondecreasing(sequenceParts);
     }
-   }
- }
+  }
+
+/**
```
Comment (AlexanderSoroka) [inline IncreasingSequence.java]: indentation is wrong

**Pull request merged**